modules/exploits/multi/http/motioneye_auth_rce_cve_2025_60787.rb

```diff
@@ -0,0 +1,404 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Remote Code Execution Vulnerability in MotionEye (CVE-2025-60787)',
+        'Description' => %q{
+          MotionEye is vulnerable to OS Command Injection in configuration parameters such as image_file_name.
+          Unsanitized user input is written to Motion configuration files,
+          allowing remote authenticated attackers with admin access to achieve code execution when Motion is restarted.
+
+          This vulnerability affects MotionEye versions <= 0.43.1b4 are vulnerable.
+          Successful exploitation may result in the remote code execution under the privileges
+          of the web server, potentially exposing sensitive data or disrupting survey operations.
+
+          An attacker can execute arbitrary system commands in the context of the user running the web server.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Maksim Rogov', # Metasploit Module
+          'prabhatverma47' # Vulnerability Discovery
+        ],
+        'References' => [
+          ['CVE', '2025-60787'],
```

Comment:
Looks like a vulnerability was fixed in https://github.com/motioneye-project/motioneye/releases/tag/0.43.1b4, not sure if this is the same bug or a different one though.